common/remote/hooks/libarchive/archive.cpp

```diff
@@ -608,7 +608,7 @@ class CArchiveFileHook : public CInterface, implements IContainedFileHook
 
 extern ARCHIVEFILE_API void installFileHook()
 {
-    SpinBlock b(*lock); // Probably overkill!
+    CriticalBlock b(*lock); // Probably overkill!
```

Comment:
Doesn't really matter what kind of lock it is here I suspect.
But we should default to use a CS if expect low contention/cost.
Is that your assertion?